src/components/Filters/Filters.tsx

```diff
@@ -348,10 +349,13 @@ class FiltersInner extends Component<CombinedProps, State> {
       </div>
     );
 
-    const tagsMarkup =
-      !hideTags && appliedFilters && appliedFilters.length ? (
-        <div className={styles.TagsContainer}>
-          {appliedFilters.map((filter) => {
+    const shouldHideTagsContainer =
+      !appliedFilters || appliedFilters.length < 1;
```

Comment:
```suggestion
      appliedFilters?.length < 1;
```
Haven't tested this but I'm assuming appliedFilters can be either null or empty array so I _think_ this should work

Reply:
Tests fail with the change so I guess it's almost but not equivalent 😄